gcsio/src/test/java/com/google/cloud/hadoop/gcsio/PerformanceCachingGoogleCloudStorageTest.java

```diff
@@ -56,7 +56,8 @@
           .build();
 
   private static final CreateObjectOptions CREATE_OBJECT_OPTIONS =
-      CreateObjectOptions.DEFAULT_OVERWRITE.toBuilder()
+      CreateObjectOptions.DEFAULT_OVERWRITE
+          .toBuilder()
```

Comment:
Interesting that it changes formatting to this, probably because of the old Google Java Format version? If so, I suggest that we upgrade master branch to Java 11 and use latest Google Java Format version.

Reply:
Ack